### ISSUE TITLE
High memory usage

### ISSUE THREAD

**Issue by vinaykulkarni89** [brave/brave-browser#21551]
I am using 5 tabs in brave and the memory utilization is 1.5 GB

**Comment by elieobeid7** [comment]
I totally agree with this issue, 100% - switched from Firefox to Brave because Firefox had this issue, a year or 2 later, the tables have turned apparently.

**Comment by CodersPlayground** [comment]
Using Linux in VMware and it's true I heaps of tabs open.
Memory keeps climbing until it reaches the available max memory allocated to the VM, 15GB all allocated to brave, something must be wrong, shouldn't tabs be suspended and memory released?
